sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -382,13 +382,14 @@ private[ui] class ExecutionPagedTable(
   }
 
   private def descriptionCell(execution: SQLExecutionUIData): Seq[Node] = {
+    val jobDescription = UIUtils.makeDescription(execution.description, basePath, plainText = false)
     val details = if (execution.details != null && execution.details.nonEmpty) {
       <span onclick="this.parentNode.querySelector('.stage-details').classList.toggle('collapsed')"
             class="expand-details">
         +details
       </span> ++
       <div class="stage-details collapsed">
-        <pre>{execution.description}<br></br>{execution.details}</pre>
+        <pre>{jobDescription}<br></br>{execution.details}</pre>
```

Comment:
It seems `jobDescription`  trims the `execution.description`. But for long queries we shouldn't trim the query and need to display in the description.

Reply:
Sorry, do not understand what you mean. `makeDescription` returns HTML rendering of a job or stage description. It will try to parse the string as HTML and make sure that it only contains anchors with root-relative links. Otherwise, the whole string will rendered as a simple escaped text.

Reply:
It the `details`, before this change it used to show entire query if the query is large. But after the change it seems, it seems not showing the entire sql query
Before change: 
![Screenshot from 2019-05-17 10-29-07](https://user-images.githubusercontent.com/23054875/57904103-01f4fd00-788f-11e9-9746-0f3692f64339.png)

After change:
![Screenshot from 2019-05-17 10-30-56](https://user-images.githubusercontent.com/23054875/57904112-09b4a180-788f-11e9-96db-61fb4a6df750.png)

Reply:
What is the entire sql query?

Reply:
@shahidki31 Got it, let me confirm this problem.

Reply:
@shahidki31 Could you please test the pr again? It is fixed in my local test.

Reply:
Thanks @uncleGen . Should we fix here too? 
https://github.com/apache/spark/blob/58989cd1b009aa37b442691e3d05a98d502c635d/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InMemoryFileIndex.scala#L204-L212

Reply:
@uncleGen I'd just look at this possible change too.